client/client_update.go

```diff
@@ -108,16 +108,15 @@ type UpdateResponse struct {
 	Image struct {
 		URI      string
 		Checksum string
-		YoctoID  string `json:"yocto_id"`
-		ID       string
+		Name     string `json:"artifact_name"`
```

Comment:
Alright, I have changed this, since it is what's in the spec now. But I'm not sure it should be. I've posted a comment in that ticket, so we can handle it separately.